<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event::services::EmmaClientWidget</class>
 <widget class="QWidget" name="Event::services::EmmaClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Emma Client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Export dir</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="edExportDir"/>
       </item>
       <item>
        <widget class="QPushButton" name="btChooseExportDir">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="edExportInterval">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> sec</string>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="singleStep">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Export interval</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edFileNameBase"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>File name base</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>IOF XML 3.0</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chExportStartListXml30">
        <property name="text">
         <string>Enable export start list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chExportResultsXml30">
        <property name="text">
         <string>Enable export results</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btExportStartListXml30">
          <property name="text">
           <string>Export start list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btExportResultsXml30">
          <property name="text">
           <string>Export results</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>RACOM Text</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="2" column="1">
       <widget class="QCheckBox" name="chExportStartListTxt">
        <property name="text">
         <string>Enable export start list</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="chExportFinishTxt">
        <property name="text">
         <string>Enable export results</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btExportSplitsTxt">
          <property name="text">
           <string>Export radio codes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btExportStartListTxt">
          <property name="text">
           <string>Export start list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btExportFinishTxt">
          <property name="text">
           <string>Export results</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Startlist export type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbRacomStartTxt">
          <property name="text">
           <string>Text</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbRacomStartCsv">
          <property name="text">
           <string>CSV</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Startlist CVS export is only supported by EmmaClient after 2023-08.&lt;br/&gt;Support can be verified in EmmaClient in 'New Racom Connection' dialog,&lt;br/&gt;if it contain checkbox 'Use extended csv file read (2023)'&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::MarkdownText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edExportDir</tabstop>
  <tabstop>btChooseExportDir</tabstop>
  <tabstop>edExportInterval</tabstop>
  <tabstop>edFileNameBase</tabstop>
  <tabstop>chExportStartListXml30</tabstop>
  <tabstop>chExportResultsXml30</tabstop>
  <tabstop>btExportStartListXml30</tabstop>
  <tabstop>btExportResultsXml30</tabstop>
  <tabstop>rbRacomStartTxt</tabstop>
  <tabstop>rbRacomStartCsv</tabstop>
  <tabstop>chExportStartListTxt</tabstop>
  <tabstop>chExportFinishTxt</tabstop>
  <tabstop>btExportSplitsTxt</tabstop>
  <tabstop>btExportStartListTxt</tabstop>
  <tabstop>btExportFinishTxt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
